content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,136 @@
+---
+reviewers:
+- verb
+- yujuhong
+title: Debug a Running Pod
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+This page shows how to investigate problems in a running pod using a
+debugging ephemeral container. This is useful when the container being
+debugged does not include debugging tools.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* You should be familiar with the basics of
+  [Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/).
+* The examples in this section require the `EphemeralContainers` [Feature
+  Gate](/docs/reference/command-line-tools-reference/feature-gates.md) to be
+  enabled and kubernetes client and server version of at least 1.16.
+* Install [kubectl debug](https://github.com/verb/kubectl-debug) using the
```

Comment:
nit: `and kubernetes client and server version of at least 1.16`? Do you mean Kubernetes v1.16 or later? I am guessing that v. 1.16 is required as the state of the feature is alpha in k8s 1.16.